lib/spack/spack/user_environment.py

```diff
@@ -32,21 +32,18 @@ def prefix_inspections(platform):
 
     inspections = {
         'bin': ['PATH'],
-        'lib': ['LD_LIBRARY_PATH', 'LIBRARY_PATH'],
-        'lib64': ['LD_LIBRARY_PATH', 'LIBRARY_PATH'],
         'man': ['MANPATH'],
         'share/man': ['MANPATH'],
         'share/aclocal': ['ACLOCAL_PATH'],
-        'include': ['CPATH'],
```

Comment:
Why do you remove `include` / `CPATH` as well in this PR?

I could be discussed, whether people wanting to build packages against other packages, they should know what they're doing, etc.

But really, this is a completely different discussion. IMHO it should be in its own PR/Issue?

Reply:
This is the fallback when etc/defaults/modules.yaml is missing for some reason (which is part of spack sources). This was just an oversight. But fair, it's technically not part of the pr.